tools/rgrnastar/rg_rnaStar.xml

```diff
@@ -973,6 +980,107 @@ used: >=5 mappings => MAPQ=0; 3-4 mappings => MAPQ=1; 2 mappings => MAPQ=3. This
             <output name="splice_junctions" file="rnastar_test_genomeSAindexNbases_02.bed"/>
             <output name="mapped_reads" file="rnastar_test_mapped_reads_genomeSAindexNbases_02.bam" compare="sim_size" delta="634" />
         </test>
+        <!-- test paired-end input and outWig -->
+        <test expect_num_outputs="6">
+            <conditional name="singlePaired">
+                <param name="sPaired" value="paired" />
+                <param name="input1" value="tophat_in2.fastqsanger" ftype="fastqsanger" />
+                <param name="input2" value="tophat_in3.fastqsanger" ftype="fastqsanger" />
+            </conditional>
+            <conditional name="refGenomeSource">
+                <param name="geneSource" value="history" />
+                <param name="genomeFastaFiles" value="tophat_test.fa" />
+                <param name="genomeSAindexNbases" value="5" />
+                <conditional name="GTFconditional">
+                    <param name="GTFselect" value="with-gtf" />
+                    <param name="sjdbOverhang" value="75"/>
+                    <param name="sjdbGTFfile" value="test1.gtf" ftype="gtf"/>
+                </conditional>
+            </conditional>
+            <conditional name="quantmode_output">
+                <param name="quantMode" value="GeneCounts"/>
+            </conditional>
+            <section name="oformat">
+                <param name="outSAMattributes" value="NH,HI,AS,nM,NM,MD,jM,jI,MC,ch" />
+                <param name="outSAMmapqUnique" value="255" />
+            </section>
+            <section name="algo">
+                <conditional name="params">
+                    <param name="settingsType" value="default" />
+                </conditional>
+            </section>
+            <section name="coverage">
+                <conditional name="outWig">
+                    <param name="outWigType" value="wiggle" />
+                    <param name="outWigTypeSecondWord" value="read2"/>
+                    <param name="outWigStrand" value="false" />
+                </conditional>
+            </section>
+            <output name="output_log" file="rnastar_test.log" compare="re_match_multiline" />
+            <output name="splice_junctions" file="rnastar_test_splicejunctions_PE.bed"/>
+            <output name="mapped_reads" file="rnastar_test_mapped_reads_PE.bam" compare="sim_size" delta="634" />
+            <output name="reads_per_gene" file="tophat_test_reads_per_gene_PE.txt" />
+            <output name="signal_unique_str1" file="tophat_Signal.Unique.both.read2.out.wig" />
+            <output name="signal_uniquemultiple_str1" file="tophat_Signal.Unique.both.read2.out.wig" />
```

Comment:
Can you check for the ftype of the outputs in order to check if setting it works.

Reply:
Good point